docs/zulipbot-usage.md

```diff
@@ -53,8 +53,9 @@ to find unclaimed issues by adding one of the following filters to your search:
     Issues labeled with the **in progress** label and/or assigned to other users have
     already been claimed.
 
-* **Collaborate in area label teams** - Receive notifications on issues and
-pull requests within your fields of expertise on the
+* **Collaborate in area label teams**
+(only for members of Zulip organization on GitHub) — Receive notifications
```

Comment:
Maybe bold or italicize "members" to emphasize

Reply:
I think bold should work fine. I'll update the commit then?

Reply:
Sure.